pages/index.tsx

```diff
@@ -151,17 +147,10 @@ const Frontpage: NextPage = () => {
             />
           </Link>
         </SectionHeader>
-
         <GridContainer>
           {popularDataProducts.length > 0 && popularDataProducts.map((product) => (
-            <AssetCard key={product.id}>
-              <Link
-                href={{
-                  pathname: "/assets/[id]",
-                  query: { id: product.id },
-                }}
-                title={product.name}
-              >
+            <Link key={product.id} href={{ pathname: "/assets/[id]", query: { id: product.id } }} title={product.name}>
+              <Card elevation="raised">
                 <Card.Header>
                   <AssetCardTitle as="p">{product.name}</AssetCardTitle>
```

Comment:
IMHO this would be improved by using a proper heading instead of `p`

Reply:
Updated using heading